packages/design-picker/src/components/pattern-assembler-cta/index.tsx

```diff
@@ -1,23 +1,19 @@
 import { Button } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import type { FC } from 'react';
+import blankCanvasImage from 'calypso/assets/images/onboarding/blank-canvas-cta.svg'; // eslint-disable-line no-restricted-imports
```

Comment:
Hmm, I think `packages/` is not supposed to import/depend from `client/`? CMIIW @arthur791004